pkgs/tools/security/osv-scanner/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
+}:
+buildGoModule rec {
+  pname = "osv-scanner";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AftJS8IdKc1oDJfDTB/HMLHu7EAVnPGPR61SOlTnC2E=";
+  };
+
+  vendorSha256 = "sha256-HUgzoQuWBRnt8+lCiu9QfO1XR5EMnqVIkrL+nIMf0IA=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
```

Comment:
may be good to set the over vars too:
```suggestion
    "-X main.version=${version}"
    "-X main.commit=${version}" # this may be better as is: n/a
    "-X main.date=1970-01-01T00:00:00Z"
```